ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_fft.py

```diff
@@ -0,0 +1,34 @@
+# local
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.helpers import handle_frontend_test
+
+
+# fft
+@handle_frontend_test(
+    fn_tree="jax.numpy.fft.fft",
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=helpers.get_dtypes("float"),
+                                         array_api_dtypes=True,
+                                         shape=helpers.ints(min_value=2).
+                                         map(lambda x: tuple([x, ])))
+    .filter(
+        lambda x: "bfloat16" not in x[0]
+        and "float16" not in x[0]
+    )
+)

```

Comment:
You will need to generate and pass all function arguments, including `a`, `n`, `axis` and `norm`.